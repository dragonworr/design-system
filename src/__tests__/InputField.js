import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { InputField } from '..'

describe('InputField', () => {
  test('it renders', () => {
    const json = renderer.create(<InputField />).toJSON()
    expect(json).toMatchSnapshot()
  })
  test('it renders a form field wth a label and placeholder', () => {
    const json = renderer
      .create(
        <InputField
          icon="locked"
          label="Card number"
          id="form-field-3"
          placeholder="0000 0000 0000 0000"
        />
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
  test('it renders a form field with a color prop', () => {
    const json = renderer
      .create(
        <InputField
          id="form-field-2"
          label="Last name"
          defaultValue="Dumoulin"
          placeholder="e.g. Dumoulin"
          color="blue"
        />
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
  test('it renders a form field with an error message', () => {
    const json = renderer
      .create(
        <InputField
          icon="circleInfo"
          color="red"
          info="Please enter a valid email address"
          label="Email Address"
          defaultValue="oliver.dumoulin@priceline.c"
          id="form-field-3"
          placeholder="example@test.com"
        />
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
  test('it renders a form field with a generic message', () => {
    const json = renderer
      .create(
        <InputField
          icon="circleInfo"
          info="This is just some info"
          label="Email Address"
          id="form-field-3"
          placeholder="example@test.com"
        />
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
