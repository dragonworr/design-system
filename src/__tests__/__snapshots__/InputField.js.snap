// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputField it renders 1`] = `
.c2 {
  width: 100%;
  text-align: center;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c3 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  font-size: 16px;
  background-color: transparent;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  border-color: #d1d6db;
  box-shadow: 0 0 0 0 #d1d6db;
  margin-top: -20px;
  padding-bottom: 8px;
  padding-top: 20px;
}

.c3::-webkit-input-placeholder {
  color: #d1d6db;
}

.c3::-moz-placeholder {
  color: #d1d6db;
}

.c3:-ms-input-placeholder {
  color: #d1d6db;
}

.c3::placeholder {
  color: #d1d6db;
}

.c3::-ms-clear {
  display: none;
}

.c3:focus {
  outline: 0;
  border-color: #007aff;
  box-shadow: 0 0 0 1px #007aff;
}

.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  padding-left: 13px;
  font-size: 10px;
}

<div>
  <label
    className="c0"
    fontSize="10px"
    htmlFor={undefined}
  />
  <div
    className="c1 c2"
    width={1}
  >
    <input
      aria-describedby={undefined}
      className="c3"
      color={undefined}
      id={undefined}
      width={1}
    />
  </div>
</div>
`;

exports[`InputField it renders a form field with a color prop 1`] = `
.c2 {
  width: 100%;
  text-align: center;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c3 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  font-size: 16px;
  background-color: transparent;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  border-color: #007aff;
  box-shadow: 0 0 0 0 #007aff;
  margin-top: -20px;
  padding-bottom: 8px;
  padding-top: 20px;
}

.c3::-webkit-input-placeholder {
  color: #d1d6db;
}

.c3::-moz-placeholder {
  color: #d1d6db;
}

.c3:-ms-input-placeholder {
  color: #d1d6db;
}

.c3::placeholder {
  color: #d1d6db;
}

.c3::-ms-clear {
  display: none;
}

.c3:focus {
  outline: 0;
  border-color: #007aff;
  box-shadow: 0 0 0 1px #007aff;
}

.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  padding-left: 13px;
  font-size: 10px;
}

<div>
  <label
    className="c0"
    fontSize="10px"
    htmlFor="form-field-2"
  >
    Last name
  </label>
  <div
    className="c1 c2"
    width={1}
  >
    <input
      aria-describedby={undefined}
      className="c3"
      color="blue"
      defaultValue="Dumoulin"
      id="form-field-2"
      placeholder="e.g. Dumoulin"
      width={1}
    />
  </div>
</div>
`;

exports[`InputField it renders a form field with a generic message 1`] = `
.c2 {
  width: 100%;
  text-align: center;
}

.c4 {
  margin-left: -40px;
  margin-right: 14px;
  margin-top: -14px;
}

.c7 {
  margin-bottom: -24px;
  width: 100%;
  color: #d1d6db;
  background-color: #d1d6db;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  font-size: 10px;
  padding-left: 13px;
  padding-right: 13px;
  padding-top: 4px;
  padding-bottom: 4px;
  color: #fff;
  font-weight: 600;
}

.c5 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
}

.c3 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  font-size: 16px;
  background-color: transparent;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  border-color: #d1d6db;
  box-shadow: 0 0 0 0 #d1d6db;
  margin-top: -20px;
  padding-bottom: 8px;
  padding-right: 40px;
  padding-top: 20px;
}

.c3::-webkit-input-placeholder {
  color: #d1d6db;
}

.c3::-moz-placeholder {
  color: #d1d6db;
}

.c3:-ms-input-placeholder {
  color: #d1d6db;
}

.c3::placeholder {
  color: #d1d6db;
}

.c3::-ms-clear {
  display: none;
}

.c3:focus {
  outline: 0;
  border-color: #007aff;
  box-shadow: 0 0 0 1px #007aff;
}

.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  padding-left: 13px;
  font-size: 10px;
}

.c6 {
  border-width: 1px;
  border-radius: 0 0 2px 2px;
  border-color: #d1d6db;
  border-style: solid;
  box-shadow: 0 -1px 0 0 #d1d6db;
}

<div>
  <label
    className="c0"
    fontSize="10px"
    htmlFor="form-field-3"
  >
    Email Address
  </label>
  <div
    className="c1 c2"
    width={1}
  >
    <input
      aria-describedby="form-field-3-info"
      className="c3"
      color={undefined}
      id="form-field-3"
      placeholder="example@test.com"
      width={1}
    />
    <div
      className="c4"
    >
      <svg
        className="c5"
        color={undefined}
        fill="currentcolor"
        height={24}
        theme={
          Object {
            "bold": 600,
            "boxShadows": Array [
              "0 0 4px 0 rgba(0,0,0,0.08)",
              "0 8px 8px 0 rgba(0,0,0,0.08)",
              "0 16px 16px 0 rgba(0,0,0,0.08)",
              "0 32px 32px 0 rgba(0,0,0,0.08)",
            ],
            "breakpoints": Array [
              32,
              40,
              48,
              64,
            ],
            "colors": Object {
              "black": "#000",
              "blue": "#007aff",
              "blue0": "#cdf",
              "blue1": "#cdf",
              "blue2": "#007aff",
              "blue3": "#007aff",
              "blues": Array [
                "#cdf",
                "#cdf",
                "#007aff",
                "#007aff",
              ],
              "borderGray": "#d1d6db",
              "darkBlue": "#049",
              "darkGray": "#364049",
              "darkGreen": "#060",
              "darkOrange": "#950",
              "darkPurple": "#407",
              "darkRed": "#800",
              "gray": "#687B8E",
              "gray0": "#f0f2f4",
              "gray1": "#f0f2f4",
              "gray2": "#687B8E",
              "gray3": "#687B8E",
              "green": "#0a0",
              "green0": "#cec",
              "green1": "#cec",
              "green2": "#0a0",
              "green3": "#0a0",
              "greens": Array [
                "#cec",
                "#cec",
                "#0a0",
                "#0a0",
              ],
              "lightBlue": "#cdf",
              "lightGray": "#f0f2f4",
              "lightGreen": "#cec",
              "lightOrange": "#fec",
              "lightPurple": "#ecf",
              "lightRed": "#fcc",
              "orange": "#f90",
              "orange0": "#fec",
              "orange1": "#fec",
              "orange2": "#f90",
              "orange3": "#f90",
              "oranges": Array [
                "#fec",
                "#fec",
                "#f90",
                "#f90",
              ],
              "purple": "#70b",
              "purple0": "#ecf",
              "purple1": "#ecf",
              "purple2": "#70b",
              "purple3": "#70b",
              "purples": Array [
                "#ecf",
                "#ecf",
                "#70b",
                "#70b",
              ],
              "red": "#c00",
              "red0": "#fcc",
              "red1": "#fcc",
              "red2": "#c00",
              "red3": "#c00",
              "reds": Array [
                "#fcc",
                "#fcc",
                "#c00",
                "#c00",
              ],
              "text": "#001833",
              "white": "#fff",
            },
            "font": "'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif",
            "fontSizes": Array [
              12,
              14,
              16,
              20,
              24,
              32,
              48,
            ],
            "fontWeights": Object {
              "bold": 600,
              "regular": 400,
            },
            "maxContainerWidth": "1280px",
            "mediaQueries": Array [
              "@media screen and (min-width:32em)",
              "@media screen and (min-width:40em)",
              "@media screen and (min-width:48em)",
              "@media screen and (min-width:64em)",
            ],
            "radii": Array [
              0,
              2,
              6,
            ],
            "radius": "2px",
            "regular": 400,
            "space": Array [
              0,
              4,
              8,
              16,
              32,
              64,
              128,
            ],
          }
        }
        viewBox="0 0 20 21"
        width={24}
      >
        <path
          d="M9.135 9.808c0-.044.037-.08.081-.08h1.552c.045 0 .081.036.081.08v6.987a.081.081 0 0 1-.081.08H9.216a.081.081 0 0 1-.08-.08V9.808zM11.285 6.345a1.25 1.25 0 1 1-2.498-.001 1.25 1.25 0 0 1 2.498 0z M9.992 20.86a9.94 9.94 0 0 1-7.072-2.93 9.94 9.94 0 0 1-2.93-7.072 9.94 9.94 0 0 1 2.93-7.071A9.93 9.93 0 0 1 9.992.857a9.94 9.94 0 0 1 7.071 2.93 9.93 9.93 0 0 1 2.93 7.071 9.94 9.94 0 0 1-2.93 7.072 9.935 9.935 0 0 1-7.071 2.93zm0-18.39c-4.623 0-8.385 3.761-8.385 8.384s3.762 8.39 8.385 8.39c4.623 0 8.389-3.763 8.389-8.39 0-4.627-3.762-8.385-8.39-8.385z"
        />
      </svg>
    </div>
  </div>
  <div
    className="c6 c7"
    color="#d1d6db"
    width={1}
  >
    <div
      className="c8"
      color="white"
      fontSize={10}
      id="form-field-3-info"
    >
      This is just some info
    </div>
  </div>
</div>
`;

exports[`InputField it renders a form field with an error message 1`] = `
.c2 {
  width: 100%;
  text-align: center;
}

.c4 {
  margin-left: -40px;
  margin-right: 14px;
  margin-top: -14px;
}

.c7 {
  margin-bottom: -24px;
  width: 100%;
  color: #c00;
  background-color: #c00;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  font-size: 10px;
  padding-left: 13px;
  padding-right: 13px;
  padding-top: 4px;
  padding-bottom: 4px;
  color: #fff;
  font-weight: 600;
}

.c5 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  color: #c00;
}

.c3 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  font-size: 16px;
  background-color: transparent;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  border-color: #c00;
  box-shadow: 0 0 0 0 #c00;
  margin-top: -20px;
  padding-bottom: 8px;
  padding-right: 40px;
  padding-top: 20px;
}

.c3::-webkit-input-placeholder {
  color: #d1d6db;
}

.c3::-moz-placeholder {
  color: #d1d6db;
}

.c3:-ms-input-placeholder {
  color: #d1d6db;
}

.c3::placeholder {
  color: #d1d6db;
}

.c3::-ms-clear {
  display: none;
}

.c3:focus {
  outline: 0;
  border-color: #c00;
  box-shadow: 0 0 0 1px #c00;
}

.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  padding-left: 13px;
  font-size: 10px;
}

.c6 {
  border-width: 1px;
  border-radius: 0 0 2px 2px;
  border-color: #c00;
  border-style: solid;
  box-shadow: 0 -1px 0 0 #c00;
}

<div>
  <label
    className="c0"
    fontSize="10px"
    htmlFor="form-field-3"
  >
    Email Address
  </label>
  <div
    className="c1 c2"
    width={1}
  >
    <input
      aria-describedby="form-field-3-info"
      className="c3"
      color="red"
      defaultValue="oliver.dumoulin@priceline.c"
      id="form-field-3"
      placeholder="example@test.com"
      width={1}
    />
    <div
      className="c4"
    >
      <svg
        className="c5"
        color="red"
        fill="currentcolor"
        height={24}
        theme={
          Object {
            "bold": 600,
            "boxShadows": Array [
              "0 0 4px 0 rgba(0,0,0,0.08)",
              "0 8px 8px 0 rgba(0,0,0,0.08)",
              "0 16px 16px 0 rgba(0,0,0,0.08)",
              "0 32px 32px 0 rgba(0,0,0,0.08)",
            ],
            "breakpoints": Array [
              32,
              40,
              48,
              64,
            ],
            "colors": Object {
              "black": "#000",
              "blue": "#007aff",
              "blue0": "#cdf",
              "blue1": "#cdf",
              "blue2": "#007aff",
              "blue3": "#007aff",
              "blues": Array [
                "#cdf",
                "#cdf",
                "#007aff",
                "#007aff",
              ],
              "borderGray": "#d1d6db",
              "darkBlue": "#049",
              "darkGray": "#364049",
              "darkGreen": "#060",
              "darkOrange": "#950",
              "darkPurple": "#407",
              "darkRed": "#800",
              "gray": "#687B8E",
              "gray0": "#f0f2f4",
              "gray1": "#f0f2f4",
              "gray2": "#687B8E",
              "gray3": "#687B8E",
              "green": "#0a0",
              "green0": "#cec",
              "green1": "#cec",
              "green2": "#0a0",
              "green3": "#0a0",
              "greens": Array [
                "#cec",
                "#cec",
                "#0a0",
                "#0a0",
              ],
              "lightBlue": "#cdf",
              "lightGray": "#f0f2f4",
              "lightGreen": "#cec",
              "lightOrange": "#fec",
              "lightPurple": "#ecf",
              "lightRed": "#fcc",
              "orange": "#f90",
              "orange0": "#fec",
              "orange1": "#fec",
              "orange2": "#f90",
              "orange3": "#f90",
              "oranges": Array [
                "#fec",
                "#fec",
                "#f90",
                "#f90",
              ],
              "purple": "#70b",
              "purple0": "#ecf",
              "purple1": "#ecf",
              "purple2": "#70b",
              "purple3": "#70b",
              "purples": Array [
                "#ecf",
                "#ecf",
                "#70b",
                "#70b",
              ],
              "red": "#c00",
              "red0": "#fcc",
              "red1": "#fcc",
              "red2": "#c00",
              "red3": "#c00",
              "reds": Array [
                "#fcc",
                "#fcc",
                "#c00",
                "#c00",
              ],
              "text": "#001833",
              "white": "#fff",
            },
            "font": "'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif",
            "fontSizes": Array [
              12,
              14,
              16,
              20,
              24,
              32,
              48,
            ],
            "fontWeights": Object {
              "bold": 600,
              "regular": 400,
            },
            "maxContainerWidth": "1280px",
            "mediaQueries": Array [
              "@media screen and (min-width:32em)",
              "@media screen and (min-width:40em)",
              "@media screen and (min-width:48em)",
              "@media screen and (min-width:64em)",
            ],
            "radii": Array [
              0,
              2,
              6,
            ],
            "radius": "2px",
            "regular": 400,
            "space": Array [
              0,
              4,
              8,
              16,
              32,
              64,
              128,
            ],
          }
        }
        viewBox="0 0 20 21"
        width={24}
      >
        <path
          d="M9.135 9.808c0-.044.037-.08.081-.08h1.552c.045 0 .081.036.081.08v6.987a.081.081 0 0 1-.081.08H9.216a.081.081 0 0 1-.08-.08V9.808zM11.285 6.345a1.25 1.25 0 1 1-2.498-.001 1.25 1.25 0 0 1 2.498 0z M9.992 20.86a9.94 9.94 0 0 1-7.072-2.93 9.94 9.94 0 0 1-2.93-7.072 9.94 9.94 0 0 1 2.93-7.071A9.93 9.93 0 0 1 9.992.857a9.94 9.94 0 0 1 7.071 2.93 9.93 9.93 0 0 1 2.93 7.071 9.94 9.94 0 0 1-2.93 7.072 9.935 9.935 0 0 1-7.071 2.93zm0-18.39c-4.623 0-8.385 3.761-8.385 8.384s3.762 8.39 8.385 8.39c4.623 0 8.389-3.763 8.389-8.39 0-4.627-3.762-8.385-8.39-8.385z"
        />
      </svg>
    </div>
  </div>
  <div
    className="c6 c7"
    color="#c00"
    width={1}
  >
    <div
      className="c8"
      color="white"
      fontSize={10}
      id="form-field-3-info"
    >
      Please enter a valid email address
    </div>
  </div>
</div>
`;

exports[`InputField it renders a form field wth a label and placeholder 1`] = `
.c2 {
  width: 100%;
  text-align: center;
}

.c4 {
  margin-left: -40px;
  margin-right: 14px;
  margin-top: -14px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
}

.c3 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  font-size: 16px;
  background-color: transparent;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  border-color: #d1d6db;
  box-shadow: 0 0 0 0 #d1d6db;
  margin-top: -20px;
  padding-bottom: 8px;
  padding-right: 40px;
  padding-top: 20px;
}

.c3::-webkit-input-placeholder {
  color: #d1d6db;
}

.c3::-moz-placeholder {
  color: #d1d6db;
}

.c3:-ms-input-placeholder {
  color: #d1d6db;
}

.c3::placeholder {
  color: #d1d6db;
}

.c3::-ms-clear {
  display: none;
}

.c3:focus {
  outline: 0;
  border-color: #007aff;
  box-shadow: 0 0 0 1px #007aff;
}

.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  padding-left: 13px;
  font-size: 10px;
}

<div>
  <label
    className="c0"
    fontSize="10px"
    htmlFor="form-field-3"
  >
    Card number
  </label>
  <div
    className="c1 c2"
    width={1}
  >
    <input
      aria-describedby={undefined}
      className="c3"
      color={undefined}
      id="form-field-3"
      placeholder="0000 0000 0000 0000"
      width={1}
    />
    <div
      className="c4"
    >
      <svg
        className="c5"
        color={undefined}
        fill="currentcolor"
        height={24}
        theme={
          Object {
            "bold": 600,
            "boxShadows": Array [
              "0 0 4px 0 rgba(0,0,0,0.08)",
              "0 8px 8px 0 rgba(0,0,0,0.08)",
              "0 16px 16px 0 rgba(0,0,0,0.08)",
              "0 32px 32px 0 rgba(0,0,0,0.08)",
            ],
            "breakpoints": Array [
              32,
              40,
              48,
              64,
            ],
            "colors": Object {
              "black": "#000",
              "blue": "#007aff",
              "blue0": "#cdf",
              "blue1": "#cdf",
              "blue2": "#007aff",
              "blue3": "#007aff",
              "blues": Array [
                "#cdf",
                "#cdf",
                "#007aff",
                "#007aff",
              ],
              "borderGray": "#d1d6db",
              "darkBlue": "#049",
              "darkGray": "#364049",
              "darkGreen": "#060",
              "darkOrange": "#950",
              "darkPurple": "#407",
              "darkRed": "#800",
              "gray": "#687B8E",
              "gray0": "#f0f2f4",
              "gray1": "#f0f2f4",
              "gray2": "#687B8E",
              "gray3": "#687B8E",
              "green": "#0a0",
              "green0": "#cec",
              "green1": "#cec",
              "green2": "#0a0",
              "green3": "#0a0",
              "greens": Array [
                "#cec",
                "#cec",
                "#0a0",
                "#0a0",
              ],
              "lightBlue": "#cdf",
              "lightGray": "#f0f2f4",
              "lightGreen": "#cec",
              "lightOrange": "#fec",
              "lightPurple": "#ecf",
              "lightRed": "#fcc",
              "orange": "#f90",
              "orange0": "#fec",
              "orange1": "#fec",
              "orange2": "#f90",
              "orange3": "#f90",
              "oranges": Array [
                "#fec",
                "#fec",
                "#f90",
                "#f90",
              ],
              "purple": "#70b",
              "purple0": "#ecf",
              "purple1": "#ecf",
              "purple2": "#70b",
              "purple3": "#70b",
              "purples": Array [
                "#ecf",
                "#ecf",
                "#70b",
                "#70b",
              ],
              "red": "#c00",
              "red0": "#fcc",
              "red1": "#fcc",
              "red2": "#c00",
              "red3": "#c00",
              "reds": Array [
                "#fcc",
                "#fcc",
                "#c00",
                "#c00",
              ],
              "text": "#001833",
              "white": "#fff",
            },
            "font": "'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif",
            "fontSizes": Array [
              12,
              14,
              16,
              20,
              24,
              32,
              48,
            ],
            "fontWeights": Object {
              "bold": 600,
              "regular": 400,
            },
            "maxContainerWidth": "1280px",
            "mediaQueries": Array [
              "@media screen and (min-width:32em)",
              "@media screen and (min-width:40em)",
              "@media screen and (min-width:48em)",
              "@media screen and (min-width:64em)",
            ],
            "radii": Array [
              0,
              2,
              6,
            ],
            "radius": "2px",
            "regular": 400,
            "space": Array [
              0,
              4,
              8,
              16,
              32,
              64,
              128,
            ],
          }
        }
        viewBox="0 0 24 24"
        width={24}
      >
        <path
          d="M5,22 L5,11.859389 L19.241836,11.859389 L19.241836,22 L5,22 Z M9.17612777,10.5021972 L6.92962686,10.5021972 L6.92962686,7.34005863 C6.92962686,4.39559501 9.31726308,2 12.2519444,2 C15.1854594,2 17.5730956,4.39559501 17.5730956,7.34005863 L17.5730956,10.4764507 L15.3265947,10.4764507 L15.3265947,7.34005863 C15.3265947,5.63844889 13.947901,4.25398924 12.2519444,4.25398924 C10.5559879,4.25398924 9.17612777,5.63844889 9.17612777,7.34005863 L9.17612777,10.5021972 Z"
        />
      </svg>
    </div>
  </div>
</div>
`;
